nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/metrics/DiskMetricsSelector.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Radicalbit S.r.l.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.radicalbit.nsdb.cluster.metrics
+
+import akka.actor.Address
+import akka.cluster.metrics.{CapacityMetricsSelector, NodeMetrics}
+import io.radicalbit.nsdb.cluster.metrics.NSDbMetrics.Disk
+
+case object DiskMetricsSelector extends CapacityMetricsSelector {
+
+  /**
+    * Remaining capacity for each node. The value is between
+    * 0.0 and 1.0, where 0.0 means no remaining capacity (full
+    * utilization) and 1.0 means full remaining capacity (zero
+    * utilization).
+    */
```

Comment:
Maybe is stupid but the other metrics agree with this? So that 1.0 is good and 0.0 is bad?
For example the CPU metric is that 1.0 is 0% CPU utilization and 0.0 is 100%?

Reply:
It's anything but stupid. 
The akka implementation of the other metrics follows exactly the same convention.

To give you a wider picture, there is a method inside `CapacityMetricSelector` with the following signature

```
/**
   * Converts the capacity values to weights. The node with lowest
   * capacity gets weight 1 (lowest usable capacity is 1%) and other
   * nodes gets weights proportional to their capacity compared to
   * the node with lowest capacity.
   */
  def weights(capacity: Map[Address, Double]): Map[Address, Int] 
```

That is the convention that will be used in the node choice algorithm.